starter-code/index.html

```diff
@@ -2,30 +2,69 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
 <body>
-  Instagram
+  <div class="fondo">
 
```

Comment:
Abria sido mejor poner todo dentro de este div, in modo tal que la parte principal se quedase también en la parte superior